webapp/crashstats/settings/base.py

```diff
@@ -470,6 +465,12 @@ def filter(self, record):
 DATABASES = {"default": dj_database_url.parse(DATABASE_URL)}
 
 
+STORAGES = {
```

Comment:
This used to be `STATICFILES_STORAGE` and is now just `STORAGES` with a different structure.

https://docs.djangoproject.com/en/5.0/releases/4.2/#custom-file-storages